### DISCUSSION TITLE
Request to understand more on integration with other tools.

### DISCUSSION THREAD

**Discussion by 2796gaurav** [ShishirPatil/gorilla#74] [Q&A] [answered]
Hi, thanks for the demo video. Just wanted to understand how did you created gorilla spotlight feature and how can we create something out of it. Also i couldnt understand How we can integreate into langchain exactly.

**Comment by alonroth** [comment]
+1

**Comment by jmanhype** [comment]
i THINK IT WILL HAVE TO BE WITH A LLM

**Comment by ShishirPatil** [comment, marked as answer]
Hey @2796gaurav @alonroth @jmanhype @austinmw @tonxxd @breezewrf  Thanks for your interest. 

Here is an example integration of 🦍 Gorilla with Langchain 🦜  [colab notebook](https://colab.research.google.com/drive/11HJWR3ylG1HSE2v78W1gRK-dKkSA0pHe?usp=sharing).

Here is a Discussion thread on integrations with [Weviate](https://github.com/weaviate/weaviate/issues/3289).  

For the spotlight search, the front end is just an extension to the [gorilla-cli](https://github.com/gorilla-llm/gorilla-cli) wrapper. You can also call the CLI like this:
```
response = openai.ChatCompletion.create(
                api_base = "http://34.132.127.197:8000/v1",
                model='gorilla-bash-v0',
                temperature=0.1,
                messages=messages,
            )
```

Let us know what new integrations you would like to see!
